cmake/ElektraUninstall.cmake

```diff
@@ -28,3 +33,78 @@ foreach (file ${files})
 		message (STATUS "File $ENV{DESTDIR}${file} does not exist.")
 	endif (IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
 endforeach (file)
+
+# ===============
+# = Directories =
+# ===============
+
+function(remove_directories directories)
+	foreach (directory ${directories})
+		set (dir "$ENV{DESTDIR}${directory}")
+		if (EXISTS "${dir}")
+			message(STATUS "Uninstalling directory ${dir}")
+			exec_program (
+				"${CMAKE_COMMAND}"
+				ARGS "-E remove_directory \"${dir}\""
+				OUTPUT_VARIABLE rm_out
+				RETURN_VALUE rm_retval)
+
+			if (NOT "${rm_retval}" STREQUAL 0)
+				message (FATAL_ERROR "Problem when removing ${dir}")
+			endif (NOT "${rm_retval}" STREQUAL 0)
+		endif (EXISTS "${dir}")
+	endforeach (directory ${directories})
+endfunction(remove_directories)
+
+set (DIRECTORIES
+	"/usr/local/include/elektra"
```

Comment:
Seems to ignore CMAKE_INSTALL_PREFIX?

Reply:
I fixed it in my latest commit. Thank you for the advice.